cmd/terralist/server/flags.go

```diff
@@ -195,6 +196,9 @@ var flags = map[string]cli.Flag{
 	GitHubTeamsFlag: &cli.StringFlag{
 		Description: "The GitHub team slugs in CSV format to use for user validation.",
 	},
+	GitHubDomainFlag: &cli.StringFlag{
+		Description: "The GitHub base domain if you are using GitHub Enterprise. (default: 'github.com')",
+	},
```

Comment:
If you set the value as the default, the implementation will automatically generate the description for the flag. Also, hardcoding it in the auth provider won't be necessary anymore.

```suggestion
	GitHubDomainFlag: &cli.StringFlag{
		Description: "The GitHub base domain if you are using GitHub Enterprise.",
		DefaultValue: "github.com"
	},
```